src/user_content.py

```diff
@@ -37,6 +37,7 @@ def get(self, owner, repo, tag, name, path):
 
     config_map = {}
     for dependency in dependencies:
+      dependency['owner'] = dependency['owner'].replace('git://github.com/', '')
```

Comment:
I thought even after we 'fixed' analysis, we would still output `owner: git://github.com/owner`